### PR TITLE
added level intro

### PR DESCRIPTION
**Pull request by Keeganw3**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,14 +4,34 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="assets/css/style.css">
     <script type="module" src="assets/js/game.js" defer></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/howler/2.2.3/howler.min.js" integrity="sha512-6+YN/9o9BWrk6wSfGxQGpt3EUK6XeHi6yeHV+TYD2GR0Sj/cggRpXr1BrAQf0as6XslxomMUxXp2vIl+fv0QRA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
 
     <title>emojiman</title>
 </head>
+<div id="intro">
+    <p id="intro-text">VAMPIRE PARTY Baddies: 🧛 Collect: 🫀🩸🧄 🌞 20 points each Destination : ⚰️
```
Comment (MojosBeans100) [inline index.html:17]: could we do a id="intro-title" and id="intro-text" beneath it?  That will match up with what Andy is doing in his map JSON files.

Comment (MojosBeans100) [reply index.html:17]: looks like a good base for now, but we might need different elements with separate ids for
- title
- text
- baddies
- points
- goal/aim

**Comment by Keeganw3** [comment]
> could we do a id="intro-title" and id="intro-text" beneath it?  That will match up with what Andy is doing in his map JSON files. 

Yes I can do that but I think the request needs to be ok'd first. i was gonna add some stuff and try to fix the game's css so I can add the intro title in the next iteration. This stuff is place holder text.

**Comment by seanyoung247** [comment]
> 

You can make the changes, push again and they should be added to the pull request.

**Comment by seanyoung247** [comment]
Closed as per discussion for resubmission

**Pull request closed without merging**